index.js

```diff
@@ -1,6 +1,22 @@
 #!/usr/bin/env node
 'use strict'
 const pretty = require('@medv/prettyjson')
+const fs = require('fs')
+const path = require('path')
+const os = require('os')
+
+const configFile = '.fxrc'
+const readSyncSafe = file => {
+  try {
+    return fs.readFileSync(file, 'utf8')
+  } catch (err) {}
+}
+
+
+const configFileContent =
```

Comment:
Why does we need to require .fxrc from cwd?

Reply:
You may want to have a local `.fxrc` config ?
Tools like prettier, eslint often check both a global (in the home) and a local config that can override your global config for a specific task.

I figured it would be usefull to add, but might be wrong.